cmd/promu.go

```diff
@@ -165,3 +163,12 @@ func stringInSlice(needle string, haystack []string) bool {
 	}
 	return false
 }
+
+func hasRequiredConfigurations(configVars []string) error {
```

Comment:
Is there actually the need to check for multiple config vars at once? We could keep it simple until the need is there.

Another option is to use [variadic function](https://gobyexample.com/variadic-functions) to simplify the caller code.

```
func hasRequiredConfigurations(configVars ...string) error {
  // ...
}

hasRequiredConfigurations("config1", "config2")
```